docs/tutorials/python/profiler.md

```diff
@@ -0,0 +1,196 @@
+# Profiling MXNet Models
+
+It is often helpful to understand what operations take how much time while running a model. This helps optimize the model to run faster. In this tutorial, we will learn how to profile MXNet models to measure their running time and memory consumption using the MXNet profiler.
+
+## The incorrect way to profile
```

Comment:
This is not incorrect. You can still use wait_to_read to time the dot operation

Reply:
Agree. But I don't want to suggest wait_to_read as the recommended way to measure time taken by operations. While it might work for toy problems like this, 
- it is harder to use to measure execution time of multiple operations (requires wait_to_read both before and after the measured operation in multiple places). 
- It is hard to use to measure running time of a block inside a sequence (which is common)
- it won't work for hybrid networks.

The goal of this tutorial is to point people to a recommended way of profiling that works for almost all cases.

However I can add a note along the lines of "While it is possible to use `wait_to_read()` before and after an operation to get running time of an operation, it is not a scalable method to measure running time of multiple operations, especially in a Sequential or Hybrid network"